scripts/manifest.js

```diff
@@ -44,10 +45,22 @@ const overrides = new Map([
 
 const icons = manifest.map(icon => {
 	const id = typeof icon === 'string' ? icon : icon.symbol;
+	const fileName = `${id.toLowerCase()}.svg`;
+	const svgPath = path.join(__dirname, '..', 'svg', 'color', fileName);
```

Comment:
I think this can just be:

```js
path.resolve(__dirname, '../svg/color/', fileName);
```

No need to specify all the dirs as separate strings.